examples/02-metaballs/Metaballs.nim

```diff
@@ -475,23 +475,23 @@ proc Start(self: ExampleMetaball) =
 proc CleanUp(self: ExampleMetaball) =
     bgfx.DestroyProgram(self.m_program)
     bgfx.Shutdown()
+    sleep(3)
 
 proc Update(self: ExampleMetaball) =
     const y_pitch = DIMS
     const z_pitch = DIMS*DIMS
-    const inv_dim = 1.0'f32/DIMS-1
+    const inv_dim = 1.0'f32/(FDIMS-1)
     let stride = cast[int](s_metaballVertices_Decl.GetStride())
 
     # Set view 0 default viewport
     bgfx.SetViewRect(0, 0, 0, cast[uint16](self.m_width), cast[uint16](self.m_height))
 
     var now = GetTime()
     var last {.global.} = GetTime()
-    let frameTime: float32 = now-last
-    var time = GetTime()
+    let frameTime: float32 =  now - last
     last = now
-    const toMs = 1000.0'f64
-
+    var time = now / 1e3       # slow down a bit
```

Comment:
Read my other comment then make time equal to now without the division and comment.
`var time = now`